packages/core/src/geometry/BufferSystem.ts

```diff
@@ -77,14 +78,36 @@ export class BufferSystem implements ISystem
         gl.bindBuffer(buffer.type, glBuffer.buffer);
     }
 
+    unbind(type: BUFFER_TYPE): void
+    {
+        const { gl } = this;
+
+        gl.bindBuffer(type, null);
+    }
+
     /**
      * Binds an uniform buffer to at the given index.
      *
      * A cache is used so a buffer will not be bound again if already bound.
      * @param buffer - the buffer to bind
      * @param index - the base index to bind it to.
      */
-    bindBufferBase(buffer: Buffer, index: number): void
+    bindUniformBufferBase(buffer: Buffer, index: number): void
+    {
+        const { gl } = this;
+
+        this.bindBufferBase(gl.UNIFORM_BUFFER, buffer, index);
+    }
+
+    /**
+     * Binds an buffer base to at the given index.
+     *
+     * A cache is used so a buffer will not be bound again if already bound.
+     * @param target - the target to bind
+     * @param buffer - the buffer to bind
+     * @param index - the base index to bind it to.
+     */
+    protected bindBufferBase(target: number, buffer: Buffer, index: number): void
```

Comment:
Changing the function signature of `bindBufferBase` is a breaking change, but admittedly, like a small one. Still, I think this change makes sense but this PR to be against `next` instead (v7.0.0) instead of `dev` (v6.x). Alternatively, you can come up with other names for these methods and keep bindBufferBase the same.

Reply:
Actually I realized that I don't have to change this. Instead, I've moved it to `TransformFeedbackSystem`.

Reply:
Updated!